key_test.go

```diff
@@ -56,6 +56,16 @@ func TestMarshal(t *testing.T) {
 		t.Fatal("invalid cert pem bytes")
 	}
 
+	der, err := cert.MarshalDER()
+	if err != nil {
```

Comment:
nit: `require.NoError(t, err)` is a bit more concise

Reply:
Agreed, but require is not currently used and this matches the current codebase style